### DISCUSSION TITLE
2021/02/21/midi-jackd-linux/

### DISCUSSION THREAD

**Discussion by giscus** [jdeluyck/kcoreorg-website#170] [Blog Comments]
# 2021/02/21/midi-jackd-linux/

About a month and a bit in me learning the piano, I’ve had to rethink my  remote piano learning setup. The latency was at points really annnoying  (sometimes nearly a second) - it’s confusing when you hit a key and only hear it when you’re hitting the next key. So,  time to look for a lower-latency setup.

https://kcore.org/2021/02/21/midi-jackd-linux/

**Comment by nebomuk** [comment]
Thanks for the guide, it's very helpful for people still using Pulseaudio. I found that it's even easier nowadays with PipeWire, if your distro has a recent version of it (Ubuntu 24+ and it's derivatives like Linux Mint). Pipewire, the successor of Pulseaudio finally seems to have a latency low enough so that JACK was not even required for me to play on the piano and here the sounds out of fluidsynth without any lag.

**Reply by jdeluyck** [reply]
Good to hear! I haven't needed this setup in a while (classes are back in-person), but good to know :)
